Automattic/jetpack
# My Jetpack: compress extras product image

This PR compresses the Extras product image in order to reduce the page loading time. (61Kb vs 26KB)

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: compress extras product image

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to add Extras product page
`https://your-testing-site/wp-admin/admin.php?page=my-jetpack#/add-extras`
* Confirm you see the image right there
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/77539/155600683-8e3efc1f-bfae-47be-810d-11d66166f6c4.png">
